javascript/prototypingCharacterClasses.md

```diff
@@ -0,0 +1,56 @@
+#Prototype Your (Character) Class#
+
+Continuing in using Javascript to build D&D characters, I decided to use prototypes.
```

Comment:
Reading this in isolation, I don't know what you're continuing from. Is this part of a series or referring to a previous TIL?